aten/src/ATen/native/sparse/cuda/SparseBlasImpl.cpp

```diff
@@ -795,6 +795,56 @@ void addmm_out_sparse_csr(
   }
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~linalg_solve~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Don't use such delimiters. It should be clear from the function names what this part of the code is about.

Reply:
Agreed. I was trying to follow the pattern in some other file, but that might be old :)